.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,4 +5,6 @@ Click the `Preview` tab and select a PR template:
 - [Standard change](?expand=1&template=standard-change.md)
 - [Small change](?expand=1&template=small-change.md)
```

Comment:
How about this?

```suggestion
- [Standard change](?expand=1&template=standard-change.md): For `feat` (features) and `fix` (bug fixes)
- [Small change](?expand=1&template=small-change.md): For other types
```

Or this?

```suggestion
- [Standard change](?expand=1&template=standard-change.md)
  - Use this template for `feat` (features) and `fix` (bug fixes).
- [Small change](?expand=1&template=small-change.md)
  - Use this template for other types.
```

Reply:
The reason is that the request should be visible when people click the button.

Reply:
Great idea! Thanks!